extensions/conventional-comments/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "conventional-comments",
+  "title": "Conventional comments",
```

Comment:
**suggestion**: Use Title Case
To be consistent with the rest of Raycast, use ”Conventional Comments” as extension name